resources/views/layouts/master.blade.php

```diff
@@ -6,6 +6,12 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    @if (isset($global_vars->custom_font_kit_id) || isset($global_vars->custom_font_name))
```

Comment:
Quick question... should this be `&&` and not `||`? Could there be a scenario where one of the two above is missing and the logic passes but then when the missing item is called via the variable we'd get errors?